cmd/kubeadm/app/phases/addons/addons_test.go

```diff
@@ -86,3 +92,71 @@ func TestCompileManifests(t *testing.T) {
 		}
 	}
 }
+
+// This unit test exists because tolerations weren't decoded correctly by kuberuntime.DecodeInto
```

Comment:
This test seems weirdly misplaced, what was the original issue and why wouldn't an integration test catch this?